### ISSUE TITLE
Foundation is not linked (anywhere?)

### ISSUE THREAD

**Issue by nagisa** [rust-lang/www.rust-lang.org#1444]
### What needs to be fixed?

Nowhere in rlo is there a link to https://foundation.rust-lang.org/. I think we should add one. As that is likely the first place many will look for a link if they don't already know its at foundation.rlo. Especially since the search engines don't yet have it as a first hit when you search for rust foundation.

### Suggested Improvement

Add a link to the foundation page to the header, footer or the governance page?

**Comment by Manishearth** [comment]
Agreed, we probably should link to the foundation, I'll let the foundation decide how they want to be linked. I suspect this is part of the plan (the website was released just this week) but cc @ashleygwilliams so she can ensure this is something on their list

**Comment by yutannihilation** [comment]
Probably related, I believe these "owned by Mozilla" sentences should be fixed as well.

e.g. https://www.rust-lang.org/policies/media-guide

**Comment by Manishearth** [comment]
Yeah, that's something that needs to be worked on too. The foundation is still spinning up, so I'd expect a couple discrepancies for a few months.

**Comment by smarnach** [comment]
There's  #1607 for the media guide.

**Comment by senekor** [comment]
The Rust Foundation is now linked on the front page under the "thanks" section.

I also can't find any "owned by Mozilla" references anymore.
